magnetis/astro-native
# [AST-42] Add faq icon and fix check icon view box

# What

Add Faq icon

# Why

Complement the design system icons

# How

Create `Faq` icon with `react-native-svg`

# Sample

<img src="https://user-images.githubusercontent.com/13890272/108892094-3e21c980-75ee-11eb-8c6c-b31a89482fc3.png" height="500" width="250" />

# QA

Run storybook locally, confirm the `Faq` icon inclusion and `Check` icon view box fix





# Extra
Fix `Check` icon view box



[AST-42](https://produtomagnetis.atlassian.net/browse/AST-42)
